File: vetores/Nivel1/Questao-4.js
// Implementar um subprograma que retorna a quantidade de elementos ímpares em um vetor

function quantidadeDeImpares(array) {
    let contador = 0;

    for (let i = 0; i < array.length; i++) {
        if (array[i] % 2 != 0 ) {
            contador++;
        }
    }
    return contador;
}

let array = [1,1,1,1,1,2,3];

console.log(quantidadeDeImpares(array));

File: src/Python/Subprogramas/Questao-4.py
#Crie uma função que recebe um vetor de números reais como entrada e retorna a média dos valores contidos no vetor.

def soma_vetor(vetor): # Função que recebe um vetor de números reais como entrada e retorna a média dos valores contidos no vetor.
    soma = 0 # Variável que armazena a soma dos valores do vetor.
    for i in range(len(vetor)): # Laço de repetição que percorre todos os elementos do vetor.
        soma += vetor[i] # Soma o elemento do vetor na variável soma.
    return soma / len(vetor) # Retorna a média dos valores do vetor.

vetor = [10,10]

print(soma_vetor(vetor))
